libsolidity/interface/CompilerStack.cpp

```diff
@@ -681,6 +681,15 @@ StringMap CompilerStack::loadMissingSources(SourceUnit const& _ast, std::string
 	for (auto const& node: _ast.nodes())
 		if (ImportDirective const* import = dynamic_cast<ImportDirective*>(node.get()))
 		{
+			if (import->path().empty())
```

Comment:
I have the feeling that the error should be reported by the parser and we should only have an assertion here.

Reply:
Changed.